CPackLists.txt

```diff
@@ -0,0 +1,92 @@
+#  INSTALL( TARGETS StopWatch
+ #           ARCHIVE
+ #           LIBRARY DESTINATION lib/StopWatch
+ #           COMPONENT libraries)
+
+ #  INSTALL( FILES ${HEADER_FILES}
+ #           DESTINATION include
+ #           COMPONENT headers)
+
+
+SET(MAJOR_VERSION 1)
+IF (NOT DEFINED ${VERSION})
+   MESSAGE("Extracting git software version")
+   execute_process(COMMAND bash "-c" "git rev-list --branches HEAD | wc -l | tr -d ' ' | tr -d '\n'" OUTPUT_VARIABLE GIT_VERSION)
+   SET(MINOR_VERSION ${GIT_VERSION})
+   SET(BUILD_NUMBER 0)
+   SET(VERSION ${MAJOR_VERSION}.${MINOR_VERSION}.${BUILD_NUMBER}) 
+ENDIF()
+MESSAGE("Software Version: ${VERSION}")
+
+IF (NOT DEFINED ${CPACK_INSTALL_PREFIX})
+   SET(CPACK_INSTALL_PREFIX "/usr/local")
+ENDIF()
+
+SET(CPACK_PACKAGE_NAME LRStopWatch)
+SET(CPACK_PACKAGE_VERSION_MAJOR ${MAJOR_VERSION})
+SET(CPACK_PACKAGE_VERSION_MINOR ${MINOR_VERSION})
+SET(CPACK_PACKAGE_VERSION_PATCH ${BUILD_NUMBER})
+SET(CPACK_PACKAGE_DESCRIPTION "LRStopWatch is a library with utilities around time expiration and timing. 
```

Comment:
The CPackage will only be used by debian users.  The `StopWatch` name is already taken so that's why I had to change it, otherwise there's a conflict.